### ISSUE TITLE
The accuracy of your code

### ISSUE THREAD

**Issue by qgh1223** [gaopengcuhk/Unofficial-Pix2Seq#1]
Can your code reach the similar accuracy compared with the results in the paper?
I'm interested in your implementations.

**Comment by gaopengcuhk** [comment]
The current code is lower than the results published in the paper due to the following reason:
(1) greedy decoding vs nucleus decoding. (2) not enough training epochs. (3) the codebase use a constant learning rate while google paper uses a warmup and cos decay schedule. (4) no parameter fine-tuning on learning rate and weight decay. (5) no sequence augmentation, multiple instance augmentation, and drop-layer. 

I will update new results gradually.

**Comment by gaopengcuhk** [comment]
Please refer to https://github.com/gaopengcuhk/Stable-Pix2Seq for pix2seq with better performance. Due to constraint computation, I can not upload the checkpoint rightnow.
